tests/containers/account/AccountContainer-test.jsx

```diff
@@ -50,39 +50,30 @@ const stripModelId = (model) => {
 };
 
 describe('AccountContainer', () => {
+    afterEach(() => {
+        loadAccountSpy.reset();
+        loadFiscalYearSnapshotSpy.reset();
+    });
     it('should make an API call for the selected account on mount', async () => {
-        const mockRedux = {
-            account: mockReduxAccount
-        };
-
         const container = mount(<AccountContainer
-            submissionPeriods={[]}
             latestPeriod={{ year: 2020 }}
             match={parameters}
             setSelectedAccount={jest.fn()}
-            account={mockRedux} />);
+            account={mockReduxAccount} />);
 
         await container.instance().accountRequest.promise;
         await container.instance().fiscalYearSnapshotRequest.promise;
 
         expect(loadAccountSpy.callCount).toEqual(1);
         expect(loadFiscalYearSnapshotSpy.callCount).toEqual(1);
-
-        loadAccountSpy.reset();
-        loadFiscalYearSnapshotSpy.reset();
     });
 
     it('should make an API call when the award ID parameter changes', async () => {
```

Comment:
this test description is wrong (not your fault)

Reply:
Good catch!